Readme.md

```diff
@@ -77,6 +77,8 @@ properties for consistency.
 
 ## AST
 
+Interatively explore the AST with [http://iamdustan.com/reworkcss_ast_explorer/](http://iamdustan.com/reworkcss_ast_explorer/).
+
```

Comment:
You could use markdown's autolink syntax here: `<http://iamdustan.com/reworkcss_ast_explorer/>`.